### ISSUE TITLE
[BUG] Searching limited to 20 items

### ISSUE THREAD

**Issue by DoM1niC** [CappielloAntonio/tempo#165]
## Describe the bug
e.g. searched Artist with amount of 20 items, don't show all tracks after search

## Expected behavior
On WebUI (Navidrome) you can load more Tracks with a button 

## To Reproduce
Searching a Artist with more then 20-50 Tracks

## Environment
 - Android device: Nothing Phone 1
 - Android OS version: 14
 - App version: main

## Logs or Screenshots
![image](https://github.com/CappielloAntonio/tempo/assets/1436958/1030b6fb-55e3-42f1-aec4-e8694de48705)

**Comment by GallowsDove** [comment]
I don't believe this is a bug, as it's necessary to limit search for performance.

I believe it makes sense to request a feature to add a "show all" button that would load all songs/albums/artists.
